packages/snaps-utils/src/node.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable import/export */
```

Comment:
Why do we need to disable a lint rule here?

Reply:
There's some duplicate exports because we're re-exporting `./manifest/node` and `./virtual-file/node`, and parts of that are also re-exported in `index.ts`.